tests/integration/test_dashboards.py

```diff
@@ -1,6 +1,11 @@
+def test_get_old_cnx_repos_view(test_client):
+    response = test_client.get("/dashboards/old-cnx-repos")
+    assert "<title>Qualitas | CNX Repos Dashboard</title>" in response
+
+
 def test_get_cnx_repos_view(test_client):
     response = test_client.get("/dashboards/cnx-repos")
```

Comment:
This test adds more than 30 seconds to the test :grimacing:  This page is so slow because of so many calls to github, history.txt and pypi.  May be I shouldn't include it?

Reply:
Ok I've resolved this by caching results in json in `tests/data`.

Reply:
we could also use vcrpy. I've been meaning to use that instead of actually calling the various APIs. It seems like that would be good for the future. For this set of changes we can leave things as they are.